tags/state/README.md

```diff
@@ -0,0 +1,46 @@
+<h1 align="center">
+  <!-- Logo -->
+  <br/>
+  @marko-tags/state
+	<br/>
+
+  <!-- Stability -->
+  <a href="https://nodejs.org/api/documentation.html#documentation_stability_index">
+    <img src="https://img.shields.io/badge/stability-stable-green.svg" alt="API Stability"/>
+  </a>
+  <!-- NPM Version -->
+  <a href="https://npmjs.org/package/@marko-tags/state">
+    <img src="https://img.shields.io/npm/v/@marko-tags/state.svg" alt="NPM Version"/>
+  </a>
+  <!-- Downloads -->
+  <a href="https://npmjs.org/package/@marko-tags/state">
+    <img src="https://img.shields.io/npm/dm/@marko-tags/state.svg" alt="Downloads"/>
+  </a>
+</h1>
+
+Add state manipulation to your templates
```

Comment:
I think this should be more descriptive. Maybe something like `Create and manipulate state directly within your templates.` @mlrawlings what do you think?